pkg/models/active_node.go

```diff
@@ -2,10 +2,11 @@ package models
 
 //ActiveNode model
 type ActiveNode struct {
-	Id        int64  `json:"id"`
-	NodeId    string `json:"node_id"`
-	Heartbeat int64  `json:"heartbeat"`
-	Sequence  int32  `json:"sequence"`
+	Id           int64  `json:"id"`
```

Comment:
I agree to Mikhail's point in the email to keep id field, as it is recommended to have a surrogate PK.